doctor/admin.py

```
@@ -0,0 +1,446 @@
+"""
+تنظیمات Django Admin برای اپلیکیشن Doctor
+Django Admin Configuration for Doctor App
+"""
+
+from django.contrib import admin
+from django.utils.html import format_html
+from django.urls import reverse
```

Ruff (0.12.2)

`django.urls.reverse` imported but unused

Remove unused import: `django.urls.reverse`

(F401)